app/views/admin/deploy_groups/show.html.erb

```diff
@@ -36,12 +36,8 @@
       <%= link_to "Edit", edit_admin_deploy_group_path(@deploy_group), class: "btn btn-primary" %>
       <% if DeployGroup.enabled? %>
         <%= link_to "Create Stages",
```

Comment:
misleading now ... would be better as `Create Stages ...` or some other kind of indicator that it's not dangerous